app/screens/home/account/components/options/user_presence/index.tsx

```diff
@@ -108,7 +129,7 @@ const UserStatus = ({currentUser}: Props) => {
         bottomSheet({
             closeButtonId: 'close-set-user-status',
             renderContent,
-            snapPoints: [snapPoint, 10],
+            snapPoints: [(snapPoint + 34), 10],
```

Comment:
this is not the way to do this I'm sure....I'm guessing there is some variable to use for the bottom safe area that makes more sense than hard coding 34

Reply:
**never use 34** as a magic number, in fact this 34 refers to the bottom inset that is **only** on some devices but not all, so do use the proper safe area bottom inset instead.

Reply:
This should be addressed now